src/loadJs.js

```diff
@@ -9,10 +9,13 @@ module.exports = function loadJs(
   options: { sync?: boolean }
 ): Promise<?cosmiconfig$Result> | ?cosmiconfig$Result {
   function parseJsFile(content: ?string): ?cosmiconfig$Result {
-    if (!content) return null;
+    if (typeof content !== 'string') return null;
```

Comment:
How about changing this check to `content == null`? I think that is clearer in terms of intention.

Reply:
hmm, i thought the linter would get angry about double equals but i'll make that change if not